src/components/lyne-cta-button/lyne-cta-button.tsx

```diff
@@ -46,9 +50,11 @@ export class LyneCtaButton {
   };
 
   public render(): JSX.Element {
-    return <button class='button' onClick={this._buttonClick}>
-      <span class='label'>{this.label}</span>
-      <span class='arrow' innerHTML={ButtonIcon} />
+    const typeClass = `button button--${this.type}`;
```

Comment:
If we would decide to go with another naming (see comment above), we would have to adjust here as well.

Reply:
yes, in some other places as well, like in the interface declarations, the tests, and the stories file